src/main/scala/lasp/hapi/server/HapiServer.scala

```diff
@@ -11,6 +12,12 @@ import org.http4s._
 import org.http4s.server.middleware.{CORS, CORSConfig}
```

Comment:
Any votes for or against this style of import (vs "_" vs separate lines)? I'm not fond of it but I could get used to it.

Reply:
I usually prefer multiple lines. However, in this case they are very closely related things, so I guess I went ahead and imported both on the same line.

Perhaps I figured that I wouldn't comment one out without the other. I don't apply this logic consistently.